x-pack/plugins/observability/public/pages/slos/components/grouped_slos/group_list_view.tsx

```diff
@@ -58,30 +76,103 @@ export function GroupListView({
   };
 
   const groupTitle = `${groupName} (${total})`;
+  const worstSLI = useSloFormattedSLIValue(summary.worst);
+  const totalViolated =
+    !isNullOrUndefined(summary.violated) && !isNullOrUndefined(summary.total)
+      ? `${summary.violated}/${summary.total}`
+      : null;
 
   return (
-    <EuiPanel>
-      <MemoEuiAccordion buttonContent={groupTitle} id={group} initialIsOpen={false}>
-        <>
-          <SlosView
-            sloList={results}
-            loading={isLoading || isRefetching}
-            error={isError}
-            isCompact={isCompact}
-            sloView={sloView}
-            group={group}
-          />
-
-          <EuiTablePagination
-            pageCount={Math.ceil(total / ITEMS_PER_PAGE)}
-            activePage={page}
-            onChangePage={handlePageClick}
-            itemsPerPage={ITEMS_PER_PAGE}
-          />
-        </>
-      </MemoEuiAccordion>
+    <EuiPanel hasBorder={true}>
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <MemoEuiAccordion
+            buttonContent={
+              <EuiFlexGroup direction="column" gutterSize="xs">
+                <EuiFlexItem>
+                  <EuiTitle size="xs">
+                    <h3>{groupTitle}</h3>
+                  </EuiTitle>
+                </EuiFlexItem>
+                {totalViolated && (
+                  <EuiFlexItem>
+                    <span>
+                      <EuiBadge color="danger">
+                        {i18n.translate('xpack.observability.slo.group.totalViolated', {
+                          defaultMessage: '{total} Violated',
+                          values: {
+                            total: totalViolated,
+                          },
+                        })}
+                      </EuiBadge>
+                    </span>
+                  </EuiFlexItem>
+                )}
+              </EuiFlexGroup>
+            }
+            extraAction={
+              <EuiFlexGroup>
+                {worstSLI && (
+                  <EuiFlexItem grow={false}>
+                    <EuiStat
+                      description={i18n.translate('xpack.observability.slo.group.worst', {
+                        defaultMessage: 'Worst SLO',
+                      })}
+                      title={worstSLI}
+                      textAlign="right"
+                      titleColor="danger"
+                      titleSize="s"
+                      reverse
+                    />
+                  </EuiFlexItem>
+                )}
+                {totalViolated && (
+                  <EuiFlexItem grow={false}>
+                    <EuiStat
+                      description={i18n.translate('xpack.observability.slo.group.violated', {
+                        defaultMessage: 'Violated',
+                      })}
+                      title={totalViolated}
+                      textAlign="right"
+                      titleColor="danger"
```

Comment:
Same here for the color when `summary.total === 0` ? Maybe something to see with @maciejforcone

Reply:
Yes, we've messaged him about this for requirements.